### PR TITLE
tests: update test run doc and add some improvements 

### PR DESCRIPTION
**Pull request by dpgaspar**
### SUMMARY
Updates scripts/tests/run.sh README to pytest and adds some improvements

### ADDITIONAL INFORMATION
- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: scripts/tests/README.md [hunk 1]
```diff
@@ -24,38 +24,50 @@ so prior to using this script make sure to launch the dev containers.
...
+```$bash
+scripts/tests/run.sh --help
+```
+
 From the superset repo root directory:
 
+- Example run all tests:
+```$bash
+scripts/tests/run.sh --module tests/charts/api_tests.py
+```
```
Comment (villebro) [inline scripts/tests/README.md:41]: Would this not run a single test?

**Review by villebro — COMMENTED** [review]
One small comment, otherwise looks good

File: scripts/tests/README.md (hunk 1 above)
Comment (dpgaspar) [reply scripts/tests/README.md:41]: ups...

**Review by villebro — APPROVED** [review]
LGTM

**Pull request merged**